src/test/java/com/connexta/search/common/SearchITests.java

```diff
@@ -70,25 +81,39 @@
           .withExposedPorts(SOLR_PORT)
           .waitingFor(Wait.forHttp("/solr/" + SOLR_COLLECTION + "/admin/ping"));
 
-  @TestConfiguration
-  static class Config {
-
-    @Bean
-    public URL solrUrl() throws MalformedURLException {
-      return new URL(
-          "http",
-          solrContainer.getContainerIpAddress(),
-          solrContainer.getMappedPort(SOLR_PORT),
-          "/solr");
-    }
-  }
-
+  private static final String INDEX_ENDPOINT_BASE_URL = "/index/";
+  @Inject private QueryManager queryManager;
+  @Inject private IndexManager indexManager;
```

Comment:
✏️ Doesn't need to be addressed in this PR, but I don't think these beans should not be injected here. I think it's easier to read, maintain, and debug if all the tests in a test class consistently use only external endpoints or only beans.